### ISSUE TITLE
Next document shortcut

### ISSUE THREAD

**Issue by JPhB** [olivierkes/manuskript#1208]
Is there a shortcut to go to the next document, i.e. without having to click on the tree?

**Comment by TheJackiMonster** [comment]
I don't think so. I assume you mean when you have opened a text file inside a chapter for example to write one scene and you want to switch to the next scene afterwards?

**Comment by JPhB** [comment]
Yes, you have finished viewing or editing a text and you want to go to the next one in the hierarchy.

**Comment by TheJackiMonster** [comment]
Hmm, what you can do already is clicking on the folder/chapter instead of the file/scene. Then you should be able to switch to the text view of that and be able to view/edit the whole chapter as it was one file. Maybe that helps.

**Comment by obw** [comment]
We have this feature in the distraction free Editor, there it goes from one scene to the next, including jumping over to the next chapter! It works there like a charm:monocle_face:.

But I think there is also one thing missing: add new chapter

Here how its looks for me, when I use it:

![screenshot](https://github.com/olivierkes/manuskript/assets/3050806/9664af1e-9ca1-4ae1-8aaf-4e76d7aec84f)

And yes I have found another and for me even better way to use the distraction free Editor in my multi monitor setup, the second Monitor has the same size, but the 16:9 and not 9:16 display:grin:.

**Comment by JPhB** [comment]
Thanks. I never used Full Screen (in V7, which I'm still using, there is no navigation option).
